axelrod/strategies/axelrod_first.py

```diff
@@ -487,3 +489,71 @@ class UnnamedStrategy(Player):
     def strategy(opponent: Player) -> Action:
         r = random.uniform(3, 7) / 10
         return random_choice(r)
+
+@FinalTransformer((D, D), name_prefix=None)
+class SteinAndRapoport(Player):
+"""
```

Comment:
This is not indented on the correct level. It should look like:

```
class SteinAndRapoport(Player):
    ```
    A player who plays...
    ...
    ```